engine-comparison/runner.sh

```diff
@@ -50,37 +49,44 @@ mkdir corpus-archives
 SYNC_TO=${BENCHMARK}-${FENGINE_NAME}
 # WAIT_PERIOD should be longer than the whole loop, otherwise a sync cycle will be missed
 WAIT_PERIOD=20
+# "cycle 0" will be "time 0", and for purposes is skipped
 CYCLE=1
 
 # Now, begin
 $EXEC_CMD
-NEXT_SYNC=$(($SECONDS + $WAIT_PERIOD)) # =$SECONDS # Make beginning measurement?
-
-# This works for libfuzzer, tbd for AFL:
-while [[ ! -f crash* ]]; do
-# while [[ "infinite loop" ]]; do
+# Setting SECONDS is fine, it still gets ++ on schedule
+SECONDS=0
+NEXT_SYNC=$WAIT_PERIOD
+# This doesn't work for leaks or timeouts, as they have different names:
+# Currently only the individual copies of test-libfuzzer.sh document
+# the particulars of how to identify complete benchmarks
+while [[ ! $(ls | grep crash) ]]; do
```

Comment:
A possible full solution is to check whether the `EXEC_CMD` process is still running.

Reply:
This is a good idea, thanks!